packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -66,15 +66,15 @@ function getESGlobalKeys(maxPerfMode: boolean) {
         // 'RangeError', // Reflective
         // 'ReferenceError', // Reflective
         'RegExp',
-        // 'Set', // Remapped
+        'Set', // Remapped
 
         'String',
         'Symbol',
         // 'SyntaxError', // Reflective
         // 'TypeError', // Reflective
         // 'URIError', // Reflective
-        // 'WeakMap', // Remapped
-        // 'WeakSet', // Remapped
+        'WeakMap', // Remapped
+        'WeakSet', // Remapped
```

Comment:
Need to remove the `// Remapped` comments, since these aren't remapped anymore 👍